test_chess.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+
+import ai_engine
+from unittest.mock import Mock
+from Piece import Knight, King, Rook
+from enums import Player, GameStatus
+import chess_engine
+
+"""
+Unit tests
+"""
+
+
+@pytest.fixture
+def game_state_mock():
+    return Mock()
+
+
+# Tests for the function get_valid_peaceful_moves in class Knight
+
+def test_knight_middle_empty(game_state_mock):
+    """
+    The knight is in the middle, all the surrounding squares are empty
+    """
+    game_state_mock.get_piece = Mock(return_value=Player.EMPTY)
+    white_knight = Knight('n', 3, 4, Player.PLAYER_1)
+    result = white_knight.get_valid_peaceful_moves(game_state_mock)
+    assert set(result) == {(1, 3), (1, 5), (2, 2), (2, 6), (4, 2), (4, 6), (5, 3), (5, 5)}
```

Comment:
very well, it would have been better if you kept the values your are asserting in a variable in order for your code to be more neat.